src/components/userProfile/OverviewRow.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+
+class OverviewRow extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      about: '',
+      keyword: '',
+      personality: '',
+      loaded: false,
+    };
+  }
+
+  componentWillMount() {
+
+    //TODO: will be updated with real data
+    this.setState({
+      about: 'Always happy to talk',
+      keyword: '# motivation # prefered_skill',
+      personality: 'Listener active',
+      loaded: true,
+    });
+  }
+
+  // Render loading page
+  renderLoadingView() {
+    return (
+      <ActivityIndicator
+        animating={!this.state.loaded}
+        style={[styles.activityIndicator]}
+        size="large"
+        />
+    );
+  }
+
+  render() {
+    if (!this.state.loaded) {
+      return this.renderLoadingView();
+    } else {
+      return (
+          <View style={styles.container}>
```

Comment:
Only 2 blanks.
I think you guys need some tool to fix these tiny things like ESLint